### ISSUE TITLE
[Bug]: herd configuration data

### ISSUE THREAD

**Issue by iEXExchanger** [beyondcode/herd-community#623]
### Platform

Windows

### Operating system version

Windows 11

### System architecture

Windows

### Herd Version

1.0.6

### Bug description

I noticed that you store configuration data on the disk where the operating system is located, if you reinstall the computer, all data will be erased, including the database.

I cannot change the location of the configuration data to avoid such cases?

**Comment by mpociot** [comment]
No, you can't currently configure the path where Herd stores configuration files and service data.
In this case you would need to manually backup the .config/herd folder.
